internal/tool/prepare.go

```diff
@@ -0,0 +1,134 @@
+package tool
+
+import (
+	"bufio"
+	"os"
+	"path"
+	"strings"
+
+	codacy "github.com/codacy/codacy-engine-golang-seed/v6"
+	"github.com/samber/lo"
+)
+
+const sourceConfigFileName = ".semgrep.yaml"
```

Comment:
```suggestion
const toolConfigurationFileName = ".semgrep.yaml"
```
Why `source`?

Reply:
because it could be found inside the source folder 🤷 I think